README.md

```diff
@@ -78,9 +78,12 @@ trying to make your app localizable and multilingual, or a hacker looking for
 a project to contribute to, please do get in touch on the mailing list and the
 IRC channel.
 
- - mailing list: https://lists.mozilla.org/listinfo/tools-l10n
+ - Mozzilla Discourse: https://discourse.mozilla.org/c/fluent
  - IRC channel: [irc://irc.mozilla.org/l20n](irc://irc.mozilla.org/l20n)
 
+ Archived:
+ - mailing list: https://lists.mozilla.org/listinfo/tools-l10n 
```

Comment:
Let's remove this, please. The first post in Fluent's Discourse pays tribute and points to the list archives. I think that should be enough.